aries-site/src/layouts/content/Example/CompareUseCases.js

```diff
@@ -0,0 +1,55 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext, Text } from 'grommet';
+import { Close, Checkmark } from 'grommet-icons';
+
+export const CompareUseCases = ({ imageSrcGood, imageAltGood, captionGood, imageSrcBad, imageAltBad, captionBad }) => {
+
+    const size = useContext(ResponsiveContext);
+    let direction;
+    if (size !== 'small') {
+        direction = 'row';
+    } else direction = 'column';
+
+    let width;
+    if (size !== 'small') {
+        width = 'medium';
```

Comment:
with a `width` of `medium` wondering if you have long content how this would look